src/app/space-wizard/space-wizard.component.ts

```diff
@@ -1,11 +1,21 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
 import { Router } from '@angular/router';
 import { DummyService } from '../dummy/dummy.service';
 import { SpaceConfigurator, IWizardSteps, Wizard, WizardNavigator, ProjectType, ProjectInfo, StackInfo, PipelineInfo, PipelineStageInfo, PipelineTaskInfo, PipelineEnvironmentInfo } from '../models/wizard';
 import { Space } from '../models/space';
 import { ProcessTemplate } from '../models/process-template';
 import { Broadcaster } from '../shared/broadcaster.service';
 
+interface IModal
```

Comment:
Why is this in the the space-wizard? This is generic and should be placed outside of it, maybe in the Models folder.